spec/commands/v2/put_content_spec.rb

```diff
@@ -117,6 +117,29 @@
         expect(State.find_by!(content_item: content_item).name).to eq("draft")
         expect(UserFacingVersion.find_by!(content_item: content_item).number).to eq(6)
       end
+
+      describe "race condtitions", skip_cleaning: true do
+        after do
+          DatabaseCleaner.clean_with :truncation
```

Comment:
I've amended the commit message to explain this:

The tests for this change use the truncation
strategy to clean the database instead of the
default transaction strategy. This is because the
threads use separate connections from the
connection pool which cannot be collectively
wrapped in a single transaction.